hubmap/hubmap/urls.py

```diff
@@ -27,6 +27,7 @@
     path('admin/', admin.site.urls),
     path('api/', include('dataportal.urls')),
     path('', include('frontend.urls')),
+    path('portal/', include('portal.urls')),
```

Comment:
If the `portal` app is going to be what's serving the entire front-end, it probably should "own" the root URL namespace instead of the `frontend` app.